include/GafferSceneUI/Private/Inspector.h

```diff
@@ -284,6 +284,12 @@ class GAFFERSCENEUI_API Inspector::Result : public IECore::RefCounted
 
 		/// The inspected value that should be displayed by the UI.
 		const IECore::Object *value() const;
+
```

Comment:
Nitpick : let's move this blank line below `typedValue()`, to separate it from `source()` and associate it with `value()`.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5483/commits/a9a19f99f6e07fc88e49a5960b3cf4befb475245